packages/flutter/lib/src/painting/rounded_rectangle_border.dart

```diff
@@ -342,18 +293,7 @@ class _RoundedRectangleToCircleBorder extends OutlinedBorder {
   @override
   Path getInnerPath(Rect rect, { TextDirection? textDirection }) {
     final RRect borderRect = _adjustBorderRadius(rect, textDirection)!.toRRect(_adjustRect(rect));
-    final RRect adjustedRect;
-    switch (side.strokeAlign) {
-      case StrokeAlign.inside:
-        adjustedRect = borderRect.deflate(side.width);
-        break;
-      case StrokeAlign.center:
-        adjustedRect = borderRect.deflate(side.width / 2);
-        break;
-      case StrokeAlign.outside:
-        adjustedRect = borderRect;
-        break;
-    }
+    final RRect adjustedRect = borderRect.deflate(ui.lerpDouble(side.width, 0, side.strokeAlign)!);
```

Comment:
I think you are using an older version. The one that was merged didn't have this. We replace the lerp with strokeInset.
 
<img width="1614" alt="image" src="https://user-images.githubusercontent.com/351125/186575435-a65e9c74-9cf1-4d6e-9f56-96721a938cf1.png">

Reply:
Wait... I'm not seeing the same thing you are.

In both the [originally landed PR](https://github.com/flutter/flutter/pull/108339/files#diff-87b32418abb85e9d4e5f0c1e3b1806e0ea53d9562ac8ed866a2f76d32928251eR279), and [here](https://github.com/flutter/flutter/pull/110234/files/52cca658583860723d94f36809872e98851e7e13#diff-87b32418abb85e9d4e5f0c1e3b1806e0ea53d9562ac8ed866a2f76d32928251eR296), the line is:

```
final RRect adjustedRect = borderRect.deflate(ui.lerpDouble(side.width, 0, side.strokeAlign)!);
```

You're looking at line 100 on the right, not line 296.

The real question is: should that be changed?  It seems to work as-is.

Reply:
You're right, though, that the example is the same as the original, I just linked it into the docs, and renamed it to match the naming convention.

Reply:
Ok, forgive me. I got confused and looked in the wrong place. It was late at night 😅.

Reply:
No worries.